requirements.txt

```diff
@@ -1,6 +1,6 @@
 datashape >= 0.4.6
 numpy >= 1.7
 pandas >= 0.15.0
-toolz >= 0.7.2
+toolz == 0.7.4
```

Comment:
I got an error with toolz==0.7.2 that was fixed by upgrading to 0.7.4.  Unfortunately, I didn't document the error message.